package.json

```diff
@@ -22,6 +22,7 @@
     "nuxt": "^2.15.7",
     "nuxt-jsonld": "^1.5.3",
     "reading-time": "^1.5.0",
+    "vue": "^2.7.16",
```

Comment:
was this required? 🤔

Reply:
Actually, When I was setting this up. There was a Vue version mismatch error. That's why I had to install vue@2.7.16.

Reply:
I removed vue from the package.json. Resolved the version mismatch error as well.